### ISSUE TITLE
Should this repo be archived?

### ISSUE THREAD

**Issue by alexgleith** [opendatacube/docker-images#18]
It can be replaced with the [JupyterHub](https://github.com/opendatacube/jupyterhub/issues/6) repo, potentially.

**Comment by Kirill888** [comment]
yes, maybe even deleted, looks like https://github.com/opendatacube/cube-in-a-box is a successor?

**Comment by alexgleith** [comment]
It was a repo with a bunch of Docker images in it, but they slowly got removed as repo-specific Dockerfile were created, such as `datacube-ows`.

**Comment by Kirill888** [comment]
Image this thing build is still used by repo linked above, but It should be probably updated to use `opendatacube/jpuyterhub` as base.
